components/Breadcrumb/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import Breadcrumb from './index'
+import { Icon } from "../Icon";
+
+const sep = <span>🐉</span>;
+const customStyle = {
+  border: '2px solid red'
+}
+
+const customItemStyle = {
+  color: 'black',
+  border: '2px solid red'
+}
+
+storiesOf("Breadcrumb", module)
+  .add("Basic Breadcrumb", () => (
```

Comment:
just a small suggestion, in storybook and in unit tests, you need not repeat the word 'breadcrumb' in the descriptions, since the story is about Bradcrumb, that is already implied.

Reply:
Ahhh, yes 👍